docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

```diff
@@ -0,0 +1,67 @@
+= Upload a large binary file
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+Software packages can be uploaded on the OTA Connect Portal by following the section xref:ota-web::upload-software-ui.adoc[Upload software versions, window="_blank"] of the OTA Connect User Guide.
+
+You can also upload large binary files (up to 3 GB) using the `garage-sign` tool on the command line.
+
+*To upload a large binary file, follow these steps:*
+
+. Ensure you have the latest version of the `targets.json` file from OTA Connect by pulling the latest version to your local repository.

```

Comment:
Do we assume that users already have a local repo server ? 
Or shall we had an initialization step, i.e garage-sign init ?

Reply:
@xaviergully good point. I will also add the initialization step.